.asf.yaml

```diff
@@ -28,7 +43,22 @@ github:
     rebase:  true
 
   protected_branches:
-    main: { }
+    main:
+      required_status_checks:
+        # strict means "Require branches to be up to date before merging".
+        strict: false
+        # contexts are the names of checks that must pass
+        contexts:
+          - Scalafmt
+          - Pull Requests / Check / Headers
```

Comment:
I would recommend checking that this works in a separately. As per https://github.com/apache/incubator-pekko-sbt-paradox/pull/5#issuecomment-1414453790 otherwise we can freeze the main branch and we have to make an INFRA ticket to unfreeze it. The way I did this I pushed a new branch (ergo `test`), created a PR against `main` that only enables the `contexts` check against `test` and then make another PR against test to figure out if the check is satisfied.